agent/stats/service_connect_linux.go

```diff
@@ -199,6 +203,13 @@ func (sc *ServiceConnectStats) HasStatsBeenSent() bool {
 	return sc.sent
 }
 
+func (sc *ServiceConnectStats) GetAppnetClient() api.AppnetClient {
```

Comment:
Can we make this not exported?

Reply:
Can we instead remove this function? There is no way to modify this client nor is it expected so we don't need to lock around it. It is provided at construction time and isn't messed with after. This is just an unnecessary lock grab.

Reply:
makes sense. Removed it